src/Requests/Company/BanIndexRequest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace TruckersMP\APIClient\Requests\Company;
+
+use Psr\Http\Client\ClientExceptionInterface;
+use TruckersMP\APIClient\Collections\Company\BanCollection;
+use TruckersMP\APIClient\Exceptions\ApiErrorException;
+use TruckersMP\APIClient\Models\CompanyBan;
+use TruckersMP\APIClient\Requests\Request;
+
+class BanIndexRequest extends Request
+{
+    /**
+     * The ID of the requested company.
+     *
+     * @var int
+     */
+    protected $companyId;
```

Comment:
We probably want to update this to be `$key` so that it's consistent when #37 is merged